packages/web-components/public/index.html

```diff
@@ -26,6 +31,18 @@
           apiKey: '',
           target: document.getElementById('search-widget'),
         })
+    </script> -->
+
+    <script src="./dist/google-map-widget.js"></script>
+    <script>
+      ReapitGoogleMapWidgetComponent &&
+        new ReapitGoogleMapWidgetComponent({
+          apiKey: 'AIzaSyCsCzPm4VwBHlr7ERyNGizRWMQMNdswzSo',
```

Comment:
Please don't commit API keys, it should be injected by Rollup at compile time from Reapit Config